Makefile

```diff
@@ -15,7 +16,7 @@ get-deps:
 	go get github.com/tools/godep
 
 .PHONY: plugins
-plugins: $(LOCAL_ENI_PLUGIN_BINARY) $(LOCAL_IPAM_PLUGIN_BINARY)
+plugins: $(LOCAL_ENI_PLUGIN_BINARY) $(LOCAL_IPAM_PLUGIN_BINARY) $(LOCAL_BRIDGE_PLUGIN_BINARY)
```

Comment:
line 9, All the plugins are sharing the same version, I think each plugin should have its own version, right?